StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/Vertical Main Screen/RowButton.swift

```diff
@@ -393,18 +386,6 @@ extension RowButton: UIGestureRecognizerDelegate {
         true
     }
 
-    func gestureRecognizer(_ gestureRecognizer: UIGestureRecognizer, shouldReceive touch: UITouch) -> Bool {
```

Comment:
Hm, why don't we need this anymore? I recall needing it otherwise the accessoryView would not get taps.

Reply:
https://developer.apple.com/documentation/uikit/touches_presses_and_gestures/using_responders_and_the_responder_chain_to_handle_events touch events go in order from the deepest subviews to the parent until something handles it, so there shouldn't ever be a need to do this.  Tapping e.g. "View more" works fine.

Reply:
On your branch, when I change to to flat w/ checkmark, the "View more" button is not tappable anymore.

Reply:
Thanks for finding that! I understand we added this now, but this isn't a great solution. The real problem is we were incorrectly setting isUserInteractionEnabled to false on the stackview that contains the "View more" button. When iOS goes down the responder chain with a touch to find which view it belongs to, it skips views (and its subviews) if they have `isUserInteractionEnabled = false`.

There were some other bugs uncovered that I fixed as well. Looks like this now:

https://github.com/user-attachments/assets/685ca1f0-c8f9-4cb0-8175-45bd9b4065df

Reply:
LMK if you want to talk more about the responder chain, it's really mind opening when you grok it.